New-language-adaptation/German/deployment/deployment.ipynb

```diff
@@ -0,0 +1,421 @@
+    "\n",
+    "- An acoustic model\n",
+    "- A language model (optional)\n",
+    "- An inverse text normalization model (optional)\n",
+    "- A punctuation and capitalization model (optional)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "28eae3c8-c606-4890-a1b0-f91bcd0245fd",
+   "metadata": {},
+   "source": [
+    "## Pre requisite \n",
+    "\n",
+    "- Make sure you have access to [NGC](https://ngc.nvidia.com) to download models if you wish to use pre-trained models. Set up the [NGC CLI tool](https://docs.ngc.nvidia.com/cli) with your NGC API key.\n",
+    "\n",
+    "- Download Riva quickstart scripts to a local directory `<RIVA_QUICKSTART_DIR>`.\n",
+    "\n",
+    "- Prepare a local folder `<RIVA_MODEL_DIR>` to put/download raw Riva models to.\n",
+    "\n",
+    "- Prepare a local folder `<RIVA_REPO_DIR>` for Riva optimized and deployed models.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3ef45ad1-0aa8-496e-8ec4-8f7ee26a813d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Download Riva quickstart\n",
+    "RIVA_VERSION = \"2.1.0\"\n",
+    "\n",
+    "!ngc registry resource download-version nvidia/riva/riva_quickstart:$RIVA_VERSION"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3fc78401-940d-47df-b004-938c41c8a603",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "CURRENT_DIR = os.getcwd()\n",
+    "\n",
+    "# Note: replace this directory with the actual Riva quickstart folder\n",
+    "RIVA_QUICKSTART_DIR = CURRENT_DIR + f'/riva_quickstart_v{RIVA_VERSION}'\n",
+    "\n",
+    "RIVA_MODEL_DIR = CURRENT_DIR + '/riva_model_dir'\n",
+    "RIVA_REPO_DIR = CURRENT_DIR + '/riva_repo_dir'\n",
+    "\n",
+    "!mkdir $RIVA_MODEL_DIR\n",
+    "!mkdir $RIVA_REPO_DIR\n",
+    "\n",
+    "print(\"Riva model dir: \", RIVA_MODEL_DIR)\n",
+    "print(\"Riva repo dir: \",RIVA_REPO_DIR)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f3886360-e685-46fa-b2a0-3130e4102270",
+   "metadata": {},
+   "source": [
+    "The next step is to point the `riva_model_loc` to the local directory `RIVA_REPO_DIR` prepared in the previous step. By default, `riva_model_loc` point to a docker volume.\n",
+    "\n",
+    "To do this, open the Riva config file `config.sh` in the `RIVA_QUICKSTART_DIR`, find the line with `riva_model_loc` and point it to the absolute path of the `RIVA_REPO_DIR` directory, as printed out in the previous step.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "44ef9f89-d706-498f-9c04-2e5703802ca3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!head -n 100 $RIVA_QUICKSTART_DIR/config.sh |grep riva_model_loc"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "900294fe-505d-43cc-bf4c-0c4ba513347b",
+   "metadata": {},
+   "source": [
+    "## Bringing models\n",
```

Comment:
```suggestion
    "## Bringing Models\n",
```

Reply:
This section didn't exist in the previous tutorial.  It started with the BYO models section instead.